rest/route/user.go

```diff
@@ -586,15 +586,19 @@ func removeHiddenProjects(permissions []rolemanager.PermissionSummary) error {
 type rolesPostRequest struct {
 	// the list of roles to add for the user
 	Roles []string `json:"roles"`
```

Comment:
Can we maybe rename the variable to `AddRoles` to match `RemoveRoles`? We can keep the json `roles` for backwards compatibility.